### PR TITLE
[Excel, Word, PowerPoint] (COM to web add-in mapping) Update docs to indicate that the feature is no longer in preview.

### PR DESCRIPTION
**Pull request by kbrandl**
As part of these updates, this PR also refactors related custom functions docs to:
- remove an include file that's referenced in only one place (hence, no need for content to be in an include file)
- consolidate all 'COM add-in and XLL UDF compatibility' info in a single article

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6ef0619](https://github.com/OfficeDev/office-js-docs-pr/pull/1209/commits/6ef06194eba309dc45c7b7845e8fa722f13c0563)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/xll-compatibility-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/includes/xll-compatibility-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907311846058262-1209%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uYoc0LrJRkEHc9WvFQlcGG%2F6gCJb9BcjZ9cezid6YcY%3D&st=2019-07-31T18%3A42%3A22Z&se=2019-08-31T18%3A47%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/make-custom-functions-compatible-with-xll-udf.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
 If you have existing Excel XLLs, you can build equivalent custom functions in an Excel add-in to extend your solution features to other platforms such as online or macOS. However, Excel add-ins don't have all of the functionality available in XLLs. Depending on the functionality your solution uses, the XLL may provide a better experience than the Excel add-in custom functions in Excel on Windows.
 
-[!include[COM add-in and XLL UDF compatibility note](../includes/xll-compatibility-note.md)]
+> [!NOTE]
+> COM add-in and XLL UDF compatibility is supported by the following platforms, when connected to an Office 365 subscription:
+> - Excel on the web
+> - Excel on Windows (version 1904 or later)
+> - Excel on Mac (version 13.329 or later)
+> 
+> To use COM add-in and XLL UDF compatibility within Excel on the web, login by using either your Office 365 subscription or a [Microsoft account](https://account.microsoft.com/account). If you don't already have an Office 365 subscription, you can get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/en-us/office/dev-program).
```
Comment (AlexJerabek) [inline docs/excel/make-custom-functions-compatible-with-xll-udf.md]: Please remove the loc text from the URL.

**Comment by AlexJerabek** [comment]
Oh, one more thing: Why is this tagged as XWP? I only saw Excel mentioned.

File: docs/excel/make-custom-functions-compatible-with-xll-udf.md (hunk 1 above)
Comment (kbrandl) [reply docs/excel/make-custom-functions-compatible-with-xll-udf.md]: Good catch (this text was copied and pasted from elsewhere and I didn't notice the locale)!

**Comment by kbrandl** [comment]
@AlexJerabek thanks for the review and feedback! Re why this PR is tagged as WXP -- one of the articles updated does contain info about a feature that's relevant to all three of those hosts. i.e.,

image

**Comment by o365devx** [comment]
Docs Build status updates of commit _[af5a87e](https://github.com/OfficeDev/office-js-docs-pr/pull/1209/commits/af5a87e39916c0e60721165ee3af2ff23e410bd1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md) | :white_check_mark:Succeeded | |
[docs/excel/make-custom-functions-compatible-with-xll-udf.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/excel/make-custom-functions-compatible-with-xll-udf.md) | :white_check_mark:Succeeded | |
[docs/excel/xll-compatibility-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/excel/xll-compatibility-requirements.md) | :white_check_mark:Succeeded | |
[docs/includes/xll-compatibility-note.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/includes/xll-compatibility-note.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-remove-preview-note/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907312101233822-1209%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SWNLR7jiYMSeT8TXSOqpXnxQfvhociVQ%2F%2FOrTmWPEyc%3D&st=2019-07-31T20%3A57%3A43Z&se=2019-08-31T21%3A02%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**